README.md

```diff
@@ -115,13 +115,16 @@ You need the following permissions to run this module.
 - IAM Services
     - **IBM Cloud Activity Tracker** service
         - `Editor` platform access
-        - `Manager` service access
+        - `Writer` service access
     - **IBM Cloud Monitoring** service
         - `Editor` platform access
         - `Manager` service access
     - **IBM Cloud Object Storage** service
         - `Editor` platform access
         - `Manager` service access
+    - **KMS** service
+        - `Editor` platform access
+        - `Manager` service access
```

Comment:
I don't see why the module would need `IBM Cloud Activity Tracker`, `IBM Cloud Monitoring`, or `KMS` permissions? Its not creating any of those resources in the module. It only creates COS instance and COS buckets